### PR TITLE
8.0 translation

### PR DESCRIPTION
**Pull request by jgrandguillaume**
Hi,

My first try to activate transifex on an OCA repo. I've writtent a little doc here : https://github.com/OCA/maintainer-tools/wiki/Activate-Transifex

If something wrong, please let me know so I can update it !

Regards,

Joël

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
 This project aim to deal with modules related to manage Financial controlling and costing in a generic way. You'll find modules that:
 
  - Compute cost based on BoM
  - Compute standard and historical margin
+ - Price historization for reporting & BI
  - ...
+
+
+# Translation state
```
Comment (pedrobaeza) [inline README.md:15]: Use a lower header size with `------------------` like the template https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md

Comment (pedrobaeza) [inline README.md:15]: s/state/status

File: README.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 [![Build Status](https://travis-ci.org/OCA/margin-analysis.svg?branch=8.0)](https://travis-ci.org/OCA/margin-analysis)
 [![Coverage Status](https://img.shields.io/coveralls/OCA/margin-analysis.svg)](https://coveralls.io/r/OCA/margin-analysis?branch=8.0)
 
-Margin Analysis
-===============
+Margin Analysis / Financial controlling
```
Comment (lmignon) [inline README.md:4]: In the same time you can make the title a real title by adding some '====' above the module title. :smirk:

File: README.md [hunk 3]
```diff
@@ -1,11 +1,17 @@
 [![Build Status](https://travis-ci.org/OCA/margin-analysis.svg?branch=8.0)](https://travis-ci.org/OCA/margin-analysis)
```
Comment (pedrobaeza) [inline README.md:1]: Take the occasion to add the runbot badge: https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md

**Comment by gurneyalex** [comment]
needs rebase.

**Pull request closed without merging**